shotover-proxy/tests/cassandra_int_tests/basic_driver_tests.rs

```diff
@@ -1,37 +1,74 @@
 use crate::helpers::ShotoverManager;
-use cassandra_cpp::{stmt, Session};
 use serial_test::serial;
 use test_helpers::docker_compose::DockerCompose;
 
-use crate::cassandra_int_tests::{assert_query_result, cassandra_connection, ResultValue};
+use crate::cassandra_int_tests::cassandra_connection;
 
-fn test_create_keyspace(session: Session) {
-    assert_query_result(
-        &session,
-        stmt!(
-            "CREATE KEYSPACE IF NOT EXISTS cycling WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : 1 };"
-        ),
-        &[],
-    );
+mod keyspace {
+    use cassandra_cpp::Session;
 
-    assert_query_result(
-        &session,
-        stmt!("SELECT release_version FROM system.local"),
-        &[&[ResultValue::Varchar("3.11.10".into())]],
-    );
+    use crate::cassandra_int_tests::{
+        assert_query_result, assert_query_result_contains, run_query, ResultValue,
+    };
 
-    assert_query_result(
-        &session,
-        stmt!("SELECT keyspace_name FROM system_schema.keyspaces;"),
-        &[
-            &[ResultValue::Varchar("cycling".into())],
-            &[ResultValue::Varchar("system_auth".into())],
-            &[ResultValue::Varchar("system_schema".into())],
-            &[ResultValue::Varchar("system_distributed".into())],
-            &[ResultValue::Varchar("system".into())],
-            &[ResultValue::Varchar("system_traces".into())],
-        ],
-    );
+    fn test_create_keyspace(session: &Session) {
+        run_query(session, "CREATE KEYSPACE IF NOT EXISTS keyspace_tests_create WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : 1 };");
```

Comment:
I think we should use assert_query_result here as a sanity check

Reply:
Sanity check against what? Unwrapping on execute should catch any errors.

Reply:
against shotover returning values where it shouldnt.

Reply:
Ah good point! I'll fix that up

Reply:
Not only does protect against bugs in shotover but helps aid people reading the tests to understand the functionality of the cassandra query.

Maybe we should have something like:
```rust
pub fn run_command(session: &Session, query: &str) {
    assert_query_result(session, query, &[]);
}
pub fn assert_query_result
fn run_query
```

Reply:
Yep I ended up with something similar